File: backend/services/ComptabiliteService/Events/EntreeRecordedEvent.cs
using EventsContracts.EventsContracts;

namespace ComptabiliteService.Events
{
    public class EntreeRecordedEvent : IEntreeRecordedEvent
    {
        public string Id { get; set; }
        public double Price { get; set; }
        public int Quantity { get; set; }
        public DateTime Date { get; set; }
    }
}

File: backend/services/stockservice/src/main/java/com/api/stockservice/domain/event/EntreeEvents/EntreeRecordedEvent.java
package com.api.stockservice.domain.event.EntreeEvents;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.OffsetDateTime;
@Data
@AllArgsConstructor
@NoArgsConstructor
public class EntreeRecordedEvent {
    private String id;
    private Double price;
    private int quantity;
    private OffsetDateTime date;
}
